### ISSUE TITLE
How to increase the detector confidence?

### ISSUE THREAD

**Issue by ayanasser** [amazon-research/siam-mot#30]
Hello, 
I ran the code with my demo video and I found that it detects a lot of false positives, 
Is there any confidence threshold parameter that I could set to control that ?

**Comment by bingshuai2019** [comment]
Yes, these three thresholds (here: https://github.com/amazon-research/siam-mot/blob/main/siammot/configs/defaults.py#L54-L56) are probably what you need to tune.
